Content.Server/Whistle/Components/WhistleComponent.cs

```diff
@@ -0,0 +1,20 @@
+using Robust.Shared.Audio;
+
+namespace Content.Server.Whistle.Components
+{
+    [RegisterComponent]
+    public sealed partial class WhistleComponent : Component
+    {
+        [DataField("effect")]
+        public string? effect = "WhistleExclamation";
+
+        [DataField("sound", required: true)]
+        public SoundSpecifier Sound = default!;
+
+        [DataField("exclamateOnePerson")]
+        public bool exclamatePerson = false;
+
+        [DataField("loudWhistleDistance")]
+        public float Distance = 0;
+    }
+}
```

Comment:
All code should be predicted, this also needs documenting.

Reply:
As i see now, my solution with event is not what all mean by prediction, also i know that there is no any useful functions for spawn prediction according to Discord search.

So, maybe i can try remake my prediction with event, like it made in CardboardBoxSystem (copypaste, yeah i know) or i must keep trying?